app/src/main/java/com/example/marcinwisniewski/intivetrainingapp/movielist/view/adapter/MovieListBindingAdapter.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.marcinwisniewski.intivetrainingapp.movielist.view.adapter
+
+import android.arch.lifecycle.LiveData
+import android.databinding.BindingAdapter
+import android.support.v7.widget.RecyclerView
+import com.example.marcinwisniewski.intivetrainingapp.movielist.model.Movie
+
+@BindingAdapter("movieList")
+fun RecyclerView.setMovieListToRecyclerview(movieList: LiveData<List<Movie>>) {
+    if (adapter is MovieListAdapter) {
+        movieList.value?.let {
+            (adapter as MovieListAdapter).setMovies(it)
```

Comment:
Is this casting needed.

Reply:
movieList.value? can be avoided optional value?

Reply:
I think that this casting is needed because without it I am not able to reach methods from my implementation of MovieListAdapter

Reply:
I really tried but I was not able to avoid value? in the movieList - maybe I can add it as todo and try to handle it in next sprint